src/lib/components/Signin.svelte

```diff
@@ -0,0 +1,61 @@
+<script lang="ts">
+    import { page } from "$app/stores";
+
+    let showSignOutButton = false;
+
+    function signIn() {
+        window.location.href = "/auth/signin";
+    }
+
+    function signOut() {
+        window.location.href = "/auth/signout";
+    }
```

Comment:
There may be a "proper" way to do redirects with sveltekit, can you check?

Reply:
goto from $app/navigation

Reply:
Unless this breaks auth.js

Reply:
theres the goto function but it sometimes doesn't redirect properly